app/components/public/add-to-calender.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glimmer/component';
+import moment from 'moment';
+
+export default class AddToCalender extends Component {
+  get calender() {
+    const params = this.args.event;
+    const startparams = params.startsAt;
+    const timezone=params.timezone;
```

Comment:
Codacy found an issue: [Operator '=' must be spaced.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6403935)